doc/examples/QC2-sample-example.mzQC

```diff
@@ -43,17 +43,16 @@
                      "uri":"https://dx.doi.org/10.1021/pr201071t"
                   },
                   {
-                     "accession":"QC:0000000",
-                     "name":"QCCaclulator",
-                     "version":"0.9.0",
-                     "uri":"qccalculator.readthedocs.io"
+                     "accession": MS:1000799",
+                     "name": "custom unreleased software tool"
+                     "value": "mzqc-pylib"
```

Comment:
I think we should probably get a CV term for myzqc-pylib as well? Or rather pymzqc. And also our other implementations.